### PR TITLE
Add missing check for 9 vert shadows in LEGACY_SHADOW_IDS ifdef

### PR DESCRIPTION
**Pull request by Arceveti**
Fixes 9 vert shadows exported by Fast64

### CODE REVIEW

**Review by Reonu — APPROVED** [review]
lgtm

**Review by thecozies — APPROVED** [review]

File: src/engine/graph_node.c [hunk 1]
```diff
@@ -430,7 +430,7 @@ struct GraphNodeShadow *init_graph_node_shadow(struct AllocOnlyPool *pool,
 
     if (graphNode != NULL) {
 #ifdef LEGACY_SHADOW_IDS
-        if (shadowType == 1 || shadowType == 2 || shadowType == 99) {
+        if (shadowType == 0 || shadowType == 1 || shadowType == 2 || shadowType == 99) {
             shadowType = SHADOW_CIRCLE;
```
Comment (thecozies) [inline src/engine/graph_node.c:434]: @Arceveti can you add defines for these?
Screenshot_20220517-183210

Comment (gheskett) [reply src/engine/graph_node.c:434]: Not enums?

Comment (thecozies) [reply src/engine/graph_node.c:434]: enums are indeed better

Comment (Arceveti) [reply src/engine/graph_node.c:437]: done

**Review by thecozies — APPROVED** [review]
perfecto 🤌

**Review by Mr-Wiseguy — APPROVED** [review]

**Pull request merged**